src/main/java/org/candlepin/subscriptions/ApplicationConfiguration.java

```diff
@@ -89,34 +85,13 @@ ApplicationProperties applicationProperties() {
     return new ApplicationProperties();
   }
 
-  @Bean
-  @Qualifier("syncSubscriptionTasks")
-  @ConfigurationProperties(prefix = "rhsm-subscriptions.subscription.tasks")
-  TaskQueueProperties syncSubscriptionQueueProperties() {
-    return new TaskQueueProperties();
-  }
-
   @Bean
   @Qualifier
   @ConfigurationProperties(prefix = "rhsm-subscriptions.subscription-prune.tasks")
   TaskQueueProperties pruneSubscriptionTasks() {
     return new TaskQueueProperties();
   }
 
-  @Bean
-  @Qualifier("reconcileCapacityTasks")
-  @ConfigurationProperties(prefix = "rhsm-subscriptions.capacity.tasks")
-  TaskQueueProperties reconcileCapacityQueueProperties() {
-    return new TaskQueueProperties();
-  }
-
-  @Bean
-  @Qualifier("offeringSyncTasks")
-  @ConfigurationProperties(prefix = "rhsm-subscriptions.product.tasks")
-  TaskQueueProperties offeringSyncQueueProperties() {
-    return new TaskQueueProperties();
-  }
```

Comment:
Because we're porting the whole functionality from the monolith to swatch-contracts, so won't have both piece of services working at the same and hence don't need the feature flagging to set which one to use.